db/migrate/20160726061957_turn_default_value_to_lookup_value.rb

```diff
@@ -0,0 +1,26 @@
+class TurnDefaultValueToLookupValue < ActiveRecord::Migration
+
+  def up
+    LookupValue.transaction do
+      LookupKey.unscoped.each do |key|
+        LookupValue.create(:match => 'default', :value => key.default_value, :lookup_key_id => key.id, :use_puppet_default => key.use_puppet_default)
+      end
+    end
+
+    remove_column :lookup_keys, :default_value
+    remove_column :lookup_keys, :use_puppet_default
+  end
+
+  def down
+    add_column :lookup_keys, :default_value, :string, :limit => 255
+    add_column :lookup_keys, :use_puppet_default, :boolean
+
+    default_lookup_values = LookupValue.where(:match => 'default')
+
+    default_lookup_values.each do |lv|
+      LookupKey.find(lv.lookup_key_id).update_attributes(:default_value => lv.value, :use_puppet_default => lv.use_puppet_default)
```

Comment:
If above you usey `LookupValue.where(:match => 'default').eager_load(:lookup_key)`, then you can use `lv.lookup_key....` without creating an additional a query for each one. 
Also consider using `update_columns` instead of `update_attributes` to generate the sql without going through activerecord.